src/gatherers/accessibility.js

```diff
@@ -42,49 +42,49 @@ class Accessibility extends Gather {
 
   static _errorAccessibility(errorString) {
     return {
-      manifest: {
+      accessibility: {
```

Comment:
yeah... lol this is pretty important, 
hahah

@paullewis we should update one of the smoke tests runs to catch this specific issue